markdown/articles/2021/07/MonkeyWorld.md

```diff
@@ -0,0 +1,193 @@
+
+{U}{R} Sneak and Show plays some number of Ragavan, Nimble Pilferer in the sideboard, so be careful in Game 2 and 3, but it is not as scary as in {U}{R} Delver.
+
+{U}{W} and {U}{G} Omnitell stayed the same and are still favorable match ups for us.
+{U}{R} Sneak and Show was the strongest variant pre MH2 and stays the strongest post MH2, but still a pretty even match up.
+
+
+## Lands
+
+Additions: 
+<row variant="hand">{{!Endurance}} {{!Urza’s Saga}}</row>
+
+Lands was a very good match up for us, but it could get very tricky.
+We had build piles to play around all sorts of mana taxing hate like Sphere of Resistance or Choke, but also around {{Pyroblast}}.
+
+Since MH2 they added {{Endurance}} and {{Urza’s Saga}}. Endurance is annoying like in every other deck, but Urza’s Saga is special. It can produce a pretty decent clock and tutor for 0/1 mana hate pieces (for example {{Pithing Needle}}).
+
+It is still a good match up but it got a bit worse, like most.
+
+
+## Jeskai Standstill
+
+Additions: 
+<row variant="hand">{{!Ragavan, Nimble Pilferer}} {{!Urza’s Saga}}</row>
+This new iteration of {{Standstill}} decks took the metagame by storm, it is just behind Delver as the number 1 contender for best deck of the format.
+Luckily for us, it doesn’t pressure our life total as hard as Delver does, but can still be very annoying.
+Cards like {{Standstill}}, {{Ragavan}} or {{Stifle}} have the potential to make games very awkward to maneuver.
+My preferred strategy against this deck is to go for a fast kill if they pressure us, but slower games are possible. Especially when they try to hide behind Standstill without a real threat, their only aggressive one drop is Ragavan.
+But be careful, they play Stifle!
+
+## Madness
+Additions: 
+<row variant="hand">{{!Blazing Rootwalla}}</row>
+Madness is a new a deck, similar to HogaakVine in Vintage.
+It has some really explosive first turns and most graveyard hate is only relevant in longer games, games we want to avoid anyway.
+
+One of the most uninteractive matchups out there. Just mulligan for a fast win, and slam.
+
+
+
+But we also lost some long “loved” enemies.
+ANT has fallen of the map completely. {{Ragavan}} was a huge blow for such a discard heavy deck.
+Black control decks met a similar fate. They were already pretty dead before {{Prismatic Ending}} entered the format, but now there are no real reasons left to play anything else than a Bant based control shell.
+With Prismatic Ending white has all the good removal and green is also an absolute must. {{Endurance}} and {{Uro}} are just too important to keep up with Delver.
+
+# How we adapted
+
+As we can see, there were some major changes in some decks. Many decks got faster or got some answers to fight us post Doomsday.
+To stay on par most lists are now on the full Turbo plan and are UB for a clean and consistent mana base. Some lists dip into Grixis for {{Pyroblast}}, but BUG and Esper are rarely seen.
```

Comment:
I guess minor comment, most are on 3 PT over 4 PT from what I've seen. Depends whether the 5-6 cyclers makes it more turbo or the 4 PT...

Reply:
Afaik, everything from 2 to 4 is common.